tests/config/driverconfig/powerstore/v2.7.0/controller.yaml

```diff
@@ -118,11 +127,11 @@ spec:
                 - key: name
                   operator: In
                   values:
-                    - <DriverDefaultReleaseName>-controller
-            topologyKey: kubernetes.io/hostname
+                  - <DriverDefaultReleaseName>-controller
+            topologyKey: kubernetes.io/hostname 
       containers:
         - name: attacher
-          image: registry.k8s.io/sig-storage/csi-attacher:v3.5.0
+          image: k8s.gcr.io/sig-storage/csi-attacher:v4.2.0
```

Comment:
This should be registry.k8s.io. This was taken care for all links that were in repo earlier. Refer https://github.com/dell/csm-operator/pull/201

Please ensure to make this changes for all the files that are added new for PowerStore in this repo

Reply:
Pointed to use the suggested registry and installed the driver again.
![image](https://github.com/dell/csm-operator/assets/109620911/9eef05e6-974e-45c8-ac1e-ad541ad5ef58)

Ran cert-csi again.
![image](https://github.com/dell/csm-operator/assets/109620911/a32ce02d-974c-427f-8211-559f0b10d865)